lib/components/form/styled.ts

```diff
@@ -230,3 +231,125 @@ export const StyledLocationField = styled(LocationField)`
     }
   }
 `
+
+export const advancedPanelClassName = 'advanced-panel'
+export const mainPanelClassName = 'main-panel'
+export const transitionDuration = prefersReducedMotion ? 0 : 175
+
+const wipeOffset = 7
+
+const transitionMixin = css`
+  transition: all ${transitionDuration}ms ease-in;
+`
+
+const wipeOutMixin = (offset: number) => css`
```

Comment:
Instead of this confusing wipe transition that provides no visual hierarchy, should we consider having the advanced settings panel appear next to the existing panel? Then a slide animation would make more sense, and there would be a clear visual hierarchy

Reply:
Can you provide an example of what you mean here? I'm having trouble visualizing

Reply:
<img width="1552" alt="Screenshot 2024-07-19 at 12 31 58 PM" src="https://github.com/user-attachments/assets/91459c93-6396-4f59-a853-dde7ef53e721">
I think this change alone would fix most of the usability problems here!